scripts/smoke-test.sh

```
@@ -62,14 +62,14 @@ log_skip() { echo -e "  ${YELLOW}SKIP${NC}  $1 — $2"; ((SKIP_COUNT+=1)); }
 # (a)-(e) HTTP 200 check
 check_http_200() {
-  local label="$1" url="$2" method="${3:-GET}"
+  local label="$1" url="$2" method="${3:-GET}" extra_flags="${4:-}"
 
   echo "─── ${label} ──────────────────────────────────────"
   echo "${method} ${url}"
 
   local http_code
   http_code=$(curl -s -o /dev/null -w "%{http_code}" -X "${method}" \
-    "${url}" --max-time "${CHECK_TIMEOUT}" 2>&1 || echo "000")
+    ${extra_flags} "${url}" --max-time "${CHECK_TIMEOUT}" 2>&1 || echo "000")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)